### DISCUSSION TITLE
The Launchpad tab does not appear on the Job Details page

### DISCUSSION THREAD

**Discussion by jlloyd-widen** [dagster-io/dagster#19039] [Q&A]
I'm on version 1.5.13. When I got to any of my Job Details page I cannot see a "Launchpad" Tab. I'm trying to integrate with `dagster-dbt` and `dagster-meltano`. Not sure where to look in the code in order to debug. I can get to the Launchpad tab only if I click on "Open in Launchpad" which appears in various places in the UI, primarily related to Runs.

**Comment by hellendag** [comment]
Hi @jlloyd-widen, the Launchpad tab is not shown in all cases. It is not shown:

- If you are looking at a job snapshot page, e.g. `/snapshots/my_job_name@[hash]` instead of `/jobs/my_job_name`
- If you are looking at an asset job

For an asset job, the associated runs should not show "Open in Launchpad" as an option.

Can you confirm that the job you are looking at is neither an asset job nor a snapshot page? Thanks!

**Reply by jlloyd-widen** [reply]
> It is not shown:
> 
> * If you are looking at a job snapshot page, e.g. `/snapshots/my_job_name@[hash]` instead of `/jobs/my_job_name`
> * If you are looking at an asset job

The job I am looking at is indeed an asset job. So that makes sense why I wouldn't see it from the jobs page (url: `http://127.0.0.1:3000/locations/__init__.py/jobs/assets`).

> For an asset job, the associated runs should not show "Open in Launchpad" as an option.

This is definitely not the behavior I see. See screenshot:

![Screenshot 2024-01-05 at 2 06 28 PM](https://github.com/dagster-io/dagster/assets/82222659/619ad351-bf15-4dc0-a8b0-82693ce8075b)

**Reply by hellendag** [reply]
Ahh, I see. We just fixed this in [1.5.14](https://github.com/dagster-io/dagster/releases/tag/1.5.14):

```
[ui] The “Open in Launchpad” option on asset run pages has been updated to link to the graph of assets or asset job instead of an unusable launchpad page.
```

If you're able to upgrade, please give it a try!

**Reply by jlloyd-widen** [reply]
Yup, I can see that. Thanks!

To be clear though, I still see the option for asset jobs in the drop down of the overview tab, like so:

![Screenshot 2024-01-05 at 2 40 08 PM](https://github.com/dagster-io/dagster/assets/82222659/01b9783b-98cf-40cb-a0e7-57606fa10ece)

Is that expected? I mean it should be for me. I have to add configuration for many of my asset job runs.

**Reply by hellendag** [reply]
Yes, this looks correct for the materialization menu, and it should open up a launchpad for configuring the materialization.

**Reply by jlloyd-widen** [reply]
Ok cool. Then I guess this is now a non-issue.


Just found another spot that kind of makes sense but not completely. This is on the partitions tab. I do need to configure these runs as well but I would expect the button to say "Launch Backfill" instead of "Materialize." But this is now overlapping with another question I have outstanding: https://github.com/dagster-io/dagster/discussions/19031

![Screenshot 2024-01-05 at 2 53 30 PM](https://github.com/dagster-io/dagster/assets/82222659/62c969eb-4c90-4d1a-94ea-6ee55bd952c1)

**Reply by hellendag** [reply]
Cool, let's close out this discussion and continue on that one. :)
